src/shared/domain/repositories/activity_repository_interface.py

```diff
@@ -36,21 +36,13 @@ def get_activity(self, code:str) -> Activity:
     @abstractmethod
     def update_enrollment(self, user_id: str, code: str, new_state: ENROLLMENT_STATE) -> Enrollment:
         """
-        If the user is enrolled in the activity, updates the state of enrollment.
+        If the user_id is enrolled in the activity, updates the state of enrollment.
```

Comment:
If the user_id is enrolled in the activity

Reply:
OK